x/stakeibc/keeper/ibc_handlers.go

```diff
@@ -102,6 +102,20 @@ func (k Keeper) HandleAcknowledgement(ctx sdk.Context, modulePacket channeltypes
 	// 	case "/ibc.applications.transfer.v1.MsgTransfer":
 	// 		// TODO: Implement!
 	// 		continue
+	// 	case "cosmos.bank.v1beta1.MsgSend":
```

Comment:
What's going on with the comments here? Probably OK but I don't entirely follow

Reply:
This hasn't been implemented yet, but I've have been adding bits and pieces of pseudocode to track the updates we're going to need to make.

Tracked in TEST-28 and TEST-33

Slightly cleaner approach might be to open a draft PR and add to that, although it makes the process of adding code slightly more complex because we'd have to switch between the working PR and the draft to add comments. Going to leave this in for now because the implementation is up next